### PR TITLE
Restructure7 8 9

### PR DESCRIPTION
**Pull request by chris-little**
Restructured clauses 6,7,8,9,10.
Put Media Types into Clause 5 Conventions, with Links
Put OpenAPI3.0 into Clause 5 Conventions
Clause 6 Overview
Clause 7 Core (Common)
Clause 8 Queries, Environmental and Info resources (includes Collections)
Clause 9 General
Add lots of lost and broken links.

### CODE REVIEW

**Comment by dblodgett-usgs** [comment]
Hey Chris -- I'll pull this branch down and build it locally to review. Will also reconcile the changes I made in #103 as part of that and get a PR opened against the Restructure branch.

**Comment by dblodgett-usgs** [comment]
This PR now includes some fixes to the build that I implemented in parallel to @chris-little

File: candidate-standard/EDR-candidate-specification.adoc [hunk 1]
```diff
@@ -90,14 +90,12 @@ include::clause_6_overview.adoc[]
 
 include::clause_7_core.adoc[]
 
-include::clause_8_collections.adoc[]
+include::clause_8_queries.adoc[]
 
 include::clause_9_general.adoc[]
 
 include::clause_10_oas.adoc[]
```
Comment (dblodgett-usgs) [inline candidate-standard/EDR-candidate-specification.adoc:97]: This file was deleted so the include throws an error.

File: candidate-standard/clause_7_core.adoc [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+
 [[rc_core-section]]
 === Overview
 
+The `Core` Requirements Class defines the requirements for locating, understanding, and accessing environmental data resources.
+
 include::requirements/requirements_class_core.adoc[]
 
-The `Core` Requirements Class defines the requirements for locating, understanding, and accessing environmental data resources. The `Core` Requirements Class is presented in five sections:
+ The `Core` Requirements Class is presented in five sections:
```
Comment (dblodgett-usgs) [inline candidate-standard/clause_7_core.adoc:15]: Note that spaces in front of this line are causing some funny formatting.

File: candidate-standard/EDR-candidate-specification.adoc (hunk 1 above)
Comment (chris-little) [reply candidate-standard/EDR-candidate-specification.adoc]: Well spotted. Thought I had editted the include

File: candidate-standard/clause_7_core.adoc (hunk 2 above)
Comment (chris-little) [reply candidate-standard/clause_7_core.adoc]: I've also seen some markup failures, but couldn't fathom.

**Pull request merged**